frontend-react/src/shared/ReportStreamFooter/ReportStreamFooter.tsx

```diff
@@ -0,0 +1,125 @@
+import {
+    Identifier,
+    IdentifierMasthead,
+    IdentifierLogos,
+    IdentifierLogo,
+    IdentifierIdentity,
+    IdentifierLinks,
+    Link,
+    IdentifierGov,
+    IdentifierLinkItem,
+    IdentifierLink,
+} from "@trussworks/react-uswds";
+import classNames from "classnames";
+
+import { USExtLink } from "../../components/USLink";
+
+import styles from "./ReportStreamFooter.module.scss";
+
+export interface ReportStreamFooterProps
+    extends React.PropsWithChildren<React.HTMLAttributes<HTMLElement>> {
+    isAlternate?: boolean;
+}
+
+export const ReportStreamFooter = ({
+    isAlternate,
+    className,
+    ...props
+}: ReportStreamFooterProps) => {
+    const classnames = classNames(
```

Comment:
I don't think we should be passing down any css classes. If we have styles / functionality that are toggled on or off, they should be passed down as props.